packages/api/src/graphql/richText.ts

```diff
@@ -177,7 +179,8 @@ export function parseRichTextNode(value: unknown, path: string[] = []): RichText
       value.underline != undefined ? {underline: value.underline as boolean} : {},
       value.strikethrough != undefined ? {strikethrough: value.strikethrough as boolean} : {},
       value.superscript != undefined ? {superscript: value.superscript as boolean} : {},
-      value.subscript != undefined ? {subscript: value.subscript as boolean} : {}
+      value.subscript != undefined ? {subscript: value.subscript as boolean} : {},
+      value.fontColor != undefined ? {fontColor: value.fontColor as string} : {}
```

Comment:
I would suggest to use `color` instead of `fontColor`. I think this is more inline with the CSS property.